sdk/cognitivelanguage/azure-ai-language-questionanswering/samples/async_samples/sample_query_knowledgebase_async.py

```diff
@@ -31,32 +27,32 @@ async def sample_query_knowledgebase():
 
     endpoint = os.environ["AZURE_QUESTIONANSWERING_ENDPOINT"]
     key = os.environ["AZURE_QUESTIONANSWERING_KEY"]
-    knowledge_base_project = os.environ["AZURE_QUESTIONANSWERING_PROJECT"]
+    project = os.environ["AZURE_QUESTIONANSWERING_PROJECT"]
+    deployment = os.environ.get("AZURE_QUESTIONANSWERING_DEPLOYMENT", "production")
 
     client = QuestionAnsweringClient(endpoint, AzureKeyCredential(key))
     async with client:
-        question="How long should my Surface battery last?"
-        output = await client.get_answers(
+        question = "How long should my Surface battery last?"
+        output = await client.question_answering.get_answers(
```

Comment:
This does not look right. It should be `client.get_answers`. If you need to do this to get access to the method, I think the code generation is not happening correctly.